source/install/config-proxy-nginx.rst

```diff
@@ -85,3 +85,67 @@ NGINX is configured using a file in the ``/etc/nginx/sites-available`` directory
   By default, the Mattermost server accepts connections on port 8065 from every machine on the network. Use your firewall to deny connections on port 8065 to all machines except the machine that hosts NGINX and the machine that you use to administer Mattermost server. If you're installing on Amazon Web Services, you can use security groups to restrict access.
 
 Now that NGINX is installed and running, you can configure it to use SSL, which allows you to use HTTPS connections and the HTTP/2 protocol.
+
+NGINX FAQ
+=========
+
+1. "Why are Websocket connections returning a 403 error?"
+
+  This is likely due to a cross-origin check failing. A check is applied for websocket code to see if the Origin header is the same as the host header and it may not be.
+
+  .. code-block:: none
+
+    location ~ /api/v[0-9]+/(users/)?websocket$ {
+      proxy_pass            http://backend;
+      (...)
+      proxy_set_header      Host (your host name);
+      proxy_set_header      X-Forwarded-For $remote_addr;
+    }
+
+  You may need to add variants of the host name that clients may call. Your NGINX log will be helpful in diagnosing the problem.
+
+2. "How do I setup nginx-proxy with the mattermost-docker installation?"
```

Comment:
Should "mattermost" be capitalised here? e.g. Mattermost?

Reply:
I leave this up to you guys, in my opinion I would leave it lower case to match the repo name that it references but I can change it.

Reply:
@AndersonWebStudio I see, that's a good point. Let's keep it as `mattermost-docker`.

Also, I hope that changes I committed are okay - we often do so directly in docs repo, but I think usually devs like to take care of changes themselves.